### DISCUSSION TITLE
Player NameTag

### DISCUSSION THREAD

**Discussion by nasiridrishi** [Zrips/CMI#8126] [Enhancement]
CMI already has chat management features where you can format chat group wise, but every other server also needs to manage players nametag in same fashion along with chat formats , it does not make sense CMI not to have that feature. 

Please do consider adding something similar to what nametagedit does.

**Comment by CloudeLecaw** [comment]
Pls consider to request a more detailed request of what you want to request instead of saying "what an other plugin does"

You should write exaclly what you want or where you see a need of instead of asking Zrips to clone a whole plugin into a cmi feature...
